### ISSUE TITLE
Not working on Angular2 final

### ISSUE THREAD

**Issue by undefinederror** [pleerock/ng2-select-controls#8]
Hi,

This is not working for me on Angular2 final.

Please see plunker 
http://plnkr.co/edit/2apH96ypMIvICOoQ8H1f?p=preview

Am I doing something wrong?

Thank you

**Comment by pleerock** [comment]
I have deprecated ng2-radio-group in favour of ng2-select-controls. try to install it and try it. [samples are here](https://github.com/pleerock/ng2-radio-group/tree/master/sample)

**Comment by ezzkht** [comment]
Samples are still using Angular RC.4 style!   
==> directives: [SELECT_DIRECTIVES]  (deprecated)

**Comment by pleerock** [comment]
yes, need to update docs, thats why I sent link to samples. Would be great if someone can contribute here
